app/views/reports/_review_record.erb

```diff
@@ -0,0 +1,178 @@
+<div class="report_sub_title first_sub_title">
+  <div class="title_box">
+    <%= record.concept_id %>-<%= record.revision_id %>
+  </div>
+  <div class="title_text">
+
+  </div>
+</div>
+
+<div class="stats_area">
+
+  <section class="report_overall">
+
+    <div class="metrics_section">
+      <%= render(partial: 'title_bar', locals: {title: "RECORD METRICS"}) %>
+      <div class="surrounding_box">
+        <% if record.collection? %>
+
+          <table>
+            <tr class="header_row">
+              <td>Concept Id</td>
+              <td>Revision Id</td>
+              <td>Short Name</td>
+              <td>DAAC</td>
+              <td>Source File</td>
+              <td>Format</td>
+              <td>Review Close Date</td>
+            </tr>
+            <tr>
+              <td><%= record.concept_id.partition('-').first %></td>
+              <td><%= record.revision_id %></td>
+              <td><%= record.short_name %></td>
+              <td><%= record.daac %></td>
+              <td><%= link_to "UMM", record.umm_json_link %></td>
+              <td><%= record.format %></td>
+              <td><%= record.formatted_closed_date %></td>
+            </tr>
+          </table>
+
+          <br>
```

Comment:
I think it would be better to add some css to the table and add extra space to the bottom